src/crimson/common/errorator.h

```diff
@@ -1118,6 +1118,8 @@ inline std::error_code ec = std::error_code(ErrorV, std::generic_category());
 template <int ErrorV>
 using ct_error_code = unthrowable_wrapper<const std::error_code&, ec<ErrorV>>;
 
+constexpr int cmp_fail_error_value = 4095;
```

Comment:
Let's move this into ct_error and reference it as ct_error::cmp_fail_error_value.